### DISCUSSION TITLE
Contribution Guidance

### DISCUSSION THREAD

**Discussion by gunjjoshi** [ruxailab/RUXAILAB#317] [General]
Hello team ! I am Gunj Joshi, an open source developer, and I am looking forward to contribute to RUXAILAB.
I am particularly interested in this organisation.So, as an initial step, from which repository can I start my journey with ? Also, it would be great if I can get to know how can I proceed further to spend the entire summer working with RUXAILAB !!
@marcgc21

**Comment by marcgc21** [comment]
Hi @gunjjoshi I recommend you that you start from our main project: https://github.com/ruxailab/RUXAILAB

However, as you can present yourself into this [thread](https://github.com/ruxailab/RUXAILAB/discussions) should be great to know how to direct you into next steps. Some folks are engaged with IA activities, others on web development, other into VR/AR activities etc.

**Reply by gunjjoshi** [reply]
Thanks for the guidance, I am inclined towards web development.

**Comment by KarinePistili** [comment]
Since we presented the introduction discussion #306 , I'll be closing this issue.
